ios/Mattermost/AppDelegate.m

```diff
@@ -129,21 +129,19 @@ -(void)cleanNotificationsFromChannel:(NSString *)channelId {
 }
 
 // Required for deeplinking
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options{
```

Comment:
Why the changes in this file?

Reply:
On the SSO flow, when the user is directed to the web browser,  After completing the sign-in process, the browser shows a modal asking if the callback link to the MM App can be open.  If we pressed yes, nothing happened.    This method is required for deep linking on iOS greater than 9 and with this , the deep-linking works.

Reply:
Oh wasn't this ported from v1? if that so, then this is ok